apps/pdp/components/heroImage.js

```diff
@@ -17,16 +17,15 @@ const HeroImage = () => (
   <div className="product-image">
     <figure>
       <Image
-        src= "https://images-dynamic-arcteryx.imgix.net/F19/450x625/Alpha-SV-Jacket-Trail-Blaze.png?auto=format"
+        src="https://images-dynamic-arcteryx.imgix.net/F19/450x625/Axe-Keeper-Black.png?auto=format"
         itemType="image"
-        alt="Alpha SV Jacket Men's Trail Blaze"
-        data-colour-name="Trail"
-        data-large-image="https://images-dynamic-arcteryx.imgix.net/F19/1350x1710/Alpha-SV-Jacket-Trail-Blaze.jpg?auto=format&amp;w=1350"
-        data-small-image="https://images-dynamic-arcteryx.imgix.net/F19/450x625/Alpha-SV-Jacket-Trail-Blaze.png?auto=format"
-       />
-      <Figcaption>Alpha SV Jacket</Figcaption>
+        alt="Axe Keeper Black"
```

Comment:
awesome ron.  my plan is to work on making all the top level props available today that we need